octoprint_mrbeam/iobeam/laserhead_handler.py

```diff
@@ -652,3 +653,36 @@ def default_laserhead_max_intensity_including_correction(self):
         """
 
         return LASERHEAD_MAX_INSTENSITY_INCLUDING_CORRECTION_FALLBACK
+
+    @property
+    def current_laserhead_lifespan(self):
+        """Return the current laser head lifespan.
+
+        Returns:
+            float: Laser head lifespan
+        """
+
+        current_laserhead_properties = self._get_laserhead_properties()
+
+        # Handle the exceptions
+        if ((isinstance(current_laserhead_properties, dict) is False) or
+                ("lifespan" not in current_laserhead_properties) or
+                (isinstance(current_laserhead_properties["lifespan"], int) is False)):
+            # Apply fallback
+            self._logger.exception(
+                "Current Laserhead lifespan couldn't be retrieved, fallback to the factor value of: {}".format(
+                    self.default_laserhead_lifespan))
+            return self.default_laserhead_lifespan
+        # Reaching here means, everything looks good
+        self._logger.debug(
+            "Current Laserhead lifespan:{}".format(current_laserhead_properties["lifespan"]))
+        return current_laserhead_properties["lifespan"]
+
+    @property
+    def default_laserhead_lifespan(self):
+        """Default lifespan for laser head. To be used by other modules at init time.
+
+        Returns:
+            float: Laser head default lifespan
```

Comment:
If we expect the return value to be float, `LASERHEAD_LIFESPAN_FALLBACK` should be 40.0. Otherwise please change the docstring

Reply:
Ohh that was a copy past error, will fix it